demo.md

```diff
@@ -20,10 +20,15 @@ $ curl -sSL -o ./argo https://github.com/argoproj/argo/releases/download/v2.0.0-
 $ chmod +x argo 
 ```
 
-## 2. Install the Controller
+## 2. Install the Controller and UI
 ```
 $ argo install
 ```
+Installation command does not configure access for argo UI. Please use following command to create externally accessable Kubernetes service:
```

Comment:
Maybe also mention that user should change the namespace according to whatever they used during installation?

Reply:
done